mesos/cli.go

```diff
@@ -411,6 +459,8 @@ func handleUpdate() error {
 	flag.StringVar(&prefix, "prefix", "", "Destination topic prefix.")
 	flag.Int64Var(&channelSize, "queue.size", math.MinInt64, "Maximum number of messages that are buffered between the consumer and producer.")
 	flag.StringVar(&constraints, "constraints", "", "Constraints (hostname=like:^master$,rack=like:^1.*$).")
+	flag.StringVar(&metricsTopic, "metrics.topic", "", "Kafka topic to produce mirror maker consumer metrics to. Optional and turned off by default.")
```

Comment:
This looks good but the default topic name should be metrics_topic or metrics-topic please no "dots"